sccp/figures/common.py

```diff
@@ -460,6 +460,22 @@ def plotUMAP_ct(labels, pf2Points, ax):
         xlabel="UMAP1",
         title="Pf2-Based Decomposition: Label Cell Types")
 
+def plotCompViolins(projection_B, cell_types, component: int, ax):
```

Comment:
@pascoesean After the discussion about interpreting the weighted projections, it would make more sense to use boxplots. I also have a function called plotCmpPerCellType(weightedprojs, cmp, ax), flattenWeightedProjs, and there is an example in figure4c.py